src/Entity/Territory.php

```diff
@@ -58,6 +58,9 @@ class Territory
     #[Ignore]
     private $authorizedCodesInsee = [];
 
+    #[ORM\Column]
+    private ?bool $isAutoAffectationEnabled = null;
```

Comment:
comme c'est `not null default 0` dans la migration. on pourrait le mettre à false par défaut ?